ApplicationLibCode/FileInterface/RifReaderFmuRft.cpp

```diff
@@ -254,54 +207,81 @@ void RifReaderFmuRft::values( const RifEclipseRftAddress& rftAddress, std::vecto
 //--------------------------------------------------------------------------------------------------
 ///
 //--------------------------------------------------------------------------------------------------
-void RifReaderFmuRft::load()
+void RifReaderFmuRft::importData()
 {
-    QString errorMsg;
-
     QFileInfo fileInfo( m_filePath );
     if ( !( fileInfo.exists() && fileInfo.isDir() && fileInfo.isReadable() ) )
     {
-        errorMsg = QString( "Directory '%1' does not exist or isn't readable" ).arg( m_filePath );
+        auto errorMsg = QString( "Directory '%1' does not exist or isn't readable" ).arg( m_filePath );
         RiaLogging::error( errorMsg );
         return;
     }
 
     QDir dir( m_filePath );
 
-    WellObservationMap wellObservations = loadWellDates( dir, &errorMsg );
-    WellObservationMap validObservations;
-    if ( wellObservations.empty() )
+    auto wellDates = importWellDates( dir.absoluteFilePath( RifReaderFmuRft::wellPathFileName() ) );
+    if ( wellDates.empty() )
     {
-        if ( errorMsg.isEmpty() )
-        {
-            errorMsg = QString( "'%1' contains no valid FMU RFT data" ).arg( m_filePath );
-        }
-        RiaLogging::error( errorMsg );
+        RiaLogging::error( QString( "'%1' contains no valid FMU RFT data" ).arg( m_filePath ) );
         return;
     }
 
-    for ( auto it = wellObservations.begin(); it != wellObservations.end(); ++it )
-    {
-        const QString&      wellName           = it->first;
-        WellObservationSet& wellObservationSet = it->second;
-        QString             txtFile            = QString( "%1.txt" ).arg( wellName );
-        QString             obsFile            = QString( "%1.obs" ).arg( wellName );
+    std::map<QString, int> nameAndMeasurementCount;
 
-        if ( !readTxtFile( dir.absoluteFilePath( txtFile ), &errorMsg, &wellObservationSet ) )
+    // Find the number of well measurements for each well
+    for ( const auto& wellDate : wellDates )
+    {
+        auto it = nameAndMeasurementCount.find( wellDate.wellName );
+        if ( it == nameAndMeasurementCount.end() )
         {
-            RiaLogging::warning( errorMsg );
-            continue;
+            nameAndMeasurementCount[wellDate.wellName] = 1;
         }
```

Comment:
`  nameAndMeasurementCount[wellDate.wellName]++` should do here? If it does not exist it will be initialized to 0 and the incremented?